docs/competition/teams/process-docs/orientation-and-pit-rules.md

```diff
@@ -0,0 +1,81 @@
+---
+original:
+  authors: Sean Bedford
+  url: https://github.com/srobo/comp-team-coord-docs/ProcessDocs/orientationAndPitRules
+---
+## Welcome to Student Robotics 2017!
+
+## Pit Rules and Guidance
+The following rules must be observed at all times in the pits:
+
+1.	No battery charging. You must hand over your battery charger and spare batteries at reception. During the competition, visit Helpdesk (on the top floor, see map) to swap a discharged battery for a charged one.
+2.	No liquids except drinks to be present on the top floor. The carpet is expensive, and easily damaged by adhesives / paint / solvents.
```

Comment:
This is only relevant to newbury, we can remove this point

Reply:
this doc is a year-specific example.